travel.py

```diff
@@ -312,9 +305,9 @@ def login(tenant):
                            type: string
                            example: ["Password does not match"]
         """
-        agent = make_user_key(tenant)
+        agent = lowercase(tenant)
         req = request.get_json()
-        user = make_user_key(req['user'])
+        user = lowercase(req['user'])
```

Comment:
not a biggie, but at this point why not just use `.lower()` directly?
I think in the JS code I kept `makeKey` as it implies that it's a method for normalizing keys (even if the implementation itself is then a trivial placeholder...)

Reply:
yeah I kept it but not entirely convinced it's totally necessary, .lower() on its own would have been enough, but I think it should be fine to leave for now.